src/engine/controls/cuecontrol.cpp

```diff
@@ -23,6 +23,12 @@ static const double CUE_MODE_NUMARK = 3.0;
 static const double CUE_MODE_MIXXX_NO_BLINK = 4.0;
 static const double CUE_MODE_CUP = 5.0;
 
+typedef enum {
```

Comment:
Do not introduce new old fashioned C enums. Change this to an enum class.

Reply:
We already have the Cue::CueType enum in src/track/cue.h (which I have converted to an enum class in #2103). Let's use it for this.

Reply:
The cue type is `CUE` (since saved loops are hot cues), so it would be confusing if we used the `CueType` enum IMHO.